### PR TITLE
Convert FalsePositiveVcf filter to WorkflowBuilder.

### PR DESCRIPTION
**Pull request by tmooney**
(This follows on the heels of #1193.)

### CODE REVIEW

File: lib/perl/Genome/Model/Tools/DetectVariants2/Filter/FalsePositiveVcfBase.pm [hunk 1]
```diff
@@ -621,81 +619,50 @@ sub generate_and_run_readcounts_in_parallel {
...
-    my @errors = $workflow->validate;
-    if (@errors) {
-        $self->error_message(@errors);
-        die "Errors validating workflow\n";
+    if(my $parent_dir = Genome::WorkflowBuilder::DAG->parent_log_dir) {
+        $log_dir = $parent_dir;
     }
+    $workflow->recursively_set_log_dir($log_dir);
 
-    Genome::Sys->disconnect_default_handles;
```
Comment (davidlmorton) [inline lib/perl/Genome/Model/Tools/DetectVariants2/Filter/FalsePositiveVcfBase.pm]: Is it okay that we don't disconnect default handles anymore?

Comment (tmooney) [inline lib/perl/Genome/Model/Tools/DetectVariants2/Filter/FalsePositiveVcfBase.pm]: Yes; it's one of the things that is now inside `Genome::WorkflowBuilder::DAG` for both [Workflows](https://github.com/genome/genome/blob/e6fc9e92729a38ab9b68b07ce584ecfbc889273a/lib/perl/Genome/WorkflowBuilder/DAG.pm#L166) and [PTero executions](https://github.com/genome/genome/blob/e6fc9e92729a38ab9b68b07ce584ecfbc889273a/lib/perl/Genome/WorkflowBuilder/DAG.pm#L191).

**Comment by mkiwala** [comment]
+1 :koala:

**Comment by acoffman** [comment]
+1

**Pull request merged**